framework/decode/vulkan_stats_consumer.h

```diff
@@ -377,6 +399,30 @@ class VulkanStatsConsumer : public gfxrecon::decode::VulkanConsumer
         ++dispatch_count_;
     }
 
+    virtual void ProcessSetDevicePropertiesCommand(format::HandleId   physical_device_id,
```

Comment:
There will be one of these for every VkPhysicalDevice so this needs to handle multiple SetDevicePropertiesCommand blocks.  I have a .gfxr here with 2 of those blocks, one for an AMD discrete GPU and another for the integrated Intel GPU, so the blocks can't be assumed to have the same values.

Reply:
should be fixed now